<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyCustomerWidget</class>
 <widget class="QDialog" name="ModifyCustomerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>会员信息修改</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/MainWindow/Resources/modify_vip.png"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_25">
           <property name="font">
            <font>
             <family>华文行楷</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>手机</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tel_lineEdit">
           <property name="placeholderText">
            <string>支持模糊查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>华文行楷</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>姓名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name_lineEdit">
           <property name="placeholderText">
            <string>支持模糊查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="font">
            <font>
             <family>华文行楷</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>会员号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="vip_lineEdit">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>不支持模糊查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="search_pushButton">
         <property name="font">
          <font>
           <family>华文行楷</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>搜索</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_pushButton">
         <property name="font">
          <font>
           <family>华文行楷</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 128);</string>
       </property>
       <property name="text">
        <string>员工可修改项：电话、微信、QQ、地址、近视/散光度数； 无法修改项：会员号、姓名、性别、消费次数、积分；双击单元格修改，修改完成后点击保存即可。(10s 搜索刷新)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
